Insights/InputEntity_Host/ProcessEntropy.yaml

```diff
@@ -187,16 +187,13 @@ Insights:
     Project:   "project Title = 'Execution by User Accounts', Process = case(array_length(Process) > 1, 'Many', array_length(Process) == 1, tostring(Process[0]), 'None'), EntropyWeight = case(array_length(Weight) > 1, strcat(tostring(round(avgWeight, 3)), ' avg'), array_length(Weight) == 1, tostring(Weight[0]), 'None'), ProcessCountOnHost = iff(isempty(Process), 0, ProcessCountOnHost)"
 
  ChartQuery: 
-  Title: "Rare Processes over Time (Weight <= 75)"
+  Title: "Rare processes over time (Weight <= 75)"
```

Comment:
Weight -> weight

Reply:
Weight is a prominent word here and specifically associated with a column.  We will leave.